robot/web/web_test.go

```diff
@@ -639,8 +664,11 @@ func TestWebReconfigure(t *testing.T) {
 	arm1Position, err = arm1.EndPosition(ctx, nil)
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, arm1Position, test.ShouldResemble, newPos)
+	test.That(t, conn.Close(), test.ShouldBeNil)
```

Comment:
I'm not sure how these connections weren't leaking in CI, but they leaked for me locally as soon as I added `WithForceDirectGRPC()` (I assume WebRTC connections do some amount of their own cleanup somehow).

Reply:
I probably wouldn't have noticed this line without the callout. I appreciate these niche observations.

Definitely worth understanding the lifetimes of these objects relative to the public APIs they provide. But that's obviously not the short-term priority.